ratelimit/redis.go

```diff
@@ -121,17 +117,27 @@ func (c *clusterLimitRedis) startSpan(ctx context.Context, spanName string) func
 func (c *clusterLimitRedis) AllowContext(ctx context.Context, clearText string) bool {
 	c.metrics.IncCounter(redisMetricsPrefix + "total")
 	now := time.Now()
-	finishSpan := c.startSpan(ctx, allowSpanName)
+
+	var span opentracing.Span
+	parentSpan, ok := parentSpan(ctx)
+	if ok {
```

Comment:
minor: `if parentSpan, ok := parentSpan(ctx); ok ` to narrow `parentSpan` scope

Reply:
makes sense yes